Ballerina-Training/Ballerina-Forum-Application/back-end/bal-forum/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "tharmigan"
```

Comment:
Shouldn't we use a common org name for these samples?

Reply:
Yes, we should. This is a mistake. Will fix it 👍🏼

Reply:
Fixed via [cb0a130](https://github.com/ballerina-guides/training-content/pull/12/commits/cb0a130f50ee26523838f18c817fe8458a2bf7dc)